src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/edpWellbore/Example.rst

```diff
@@ -180,38 +177,30 @@ The parameters used in the simulation are summarized in the following table.
 | :math:`\phi_i`   | Initial Friction Angle  | [degree]         | 15.27         |
 +------------------+-------------------------+------------------+---------------+
 | :math:`\phi_r`   | Residual Friction Angle | [degree]         | 23.05         |
 +------------------+-------------------------+------------------+---------------+
-| :math:`c_h`      | Hardening Rate          | [-]              | 0.01          |
+| :math:`m`      | Hardening Rate          | [-]              | 0.01          |
 +------------------+-------------------------+------------------+---------------+
 | :math:`\sigma_h` | Horizontal Stress       | [MPa]            | -11.25        |
 +------------------+-------------------------+------------------+---------------+
 | :math:`\sigma_v` | Vertical Stress         | [MPa]            | -15.0         |
 +------------------+-------------------------+------------------+---------------+
 | :math:`a_0`      | Initial Well Radius     | [m]              | 0.1           |
 +------------------+-------------------------+------------------+---------------+
-| :math:`P_w`      | Mud Pressure            | [MPa]            | -2.0          |
+| :math:`p_w`      | Mud Pressure            | [MPa]            | -2.0          |
 +------------------+-------------------------+------------------+---------------+
 
 ---------------------------------
 Inspecting results
 ---------------------------------
 
-In the above example, we requested silo-format output files. We can therefore import these into VisIt and use python scripts to visualize the outcome. Below figure shows the comparisons between the numerical predictions (marks) and the corresponding analytical solutions (solid curves) with respect to the distributions of normal stress components, stress path, the supporting wellbore pressure and wellbore size. It is clear that the GEOS predictions are in excellent agreement with the analytical results. 
-
-
-.. _problemVerificationEDPWellboreFig:
-.. figure:: Verification.png
-   :align: center
-   :width: 1000
-   :figclass: align-center
-
-   Comparing GEOS results with analytical solutions
+In the above example, we requested hdf5 output files. We can therefore use python scripts to visualize the outcome. Below figure shows the comparisons between the numerical predictions (marks) and the corresponding analytical solutions (solid curves) with respect to the distributions of principal stress components, stress path on the wellbore surface, the supporting wellbore pressure and wellbore size. It is clear that the GEOS predictions are in excellent agreement with the analytical results. On the top-right figure, we added also a comparison between GEOS results for elasto-plastic material and the anlytical solutions of an elastic material. Note that the elastic solutions are differed from the elasto-plastic results even in the elastic zone (r/a>2).
 
+.. plot:: docs/sphinx/advancedExamples/validationStudies/wellboreProblems/edpWellbore/edpWellbore_plot.py
```

Comment:
Please also introduce this python script, when mentioning the locations of xml files at the beginning.

Reply:
done